code/drasil-example/nopcm/lib/Drasil/NoPCM/Body.hs

```diff
@@ -194,14 +195,18 @@ si = SI {
    refdb       = refDB
 }
 
+progName :: CI
+progName = commonIdeaWithDict "swhsNoPCM"   (nounPhrase "solar water heating system"
+  "solar water heating systems") "NoPCM" [materialEng]
```

Comment:
I think this should have a different "full" name than "solar water heating system"; maybe something like "solar water heating system with no phase change material" @smiths any thoughts?

Reply:
Yes, I agree with @samm82.  The full name should distinguish the nopcm example from the with pcm example, just like the short name does.  I like the suggested full name from @samm82.